### ISSUE TITLE
hi another engine from you: ? Wow!

### ISSUE THREAD

**Issue by zartan9** [haroldo-ok/retro-game-editor#19]
just saw this one ..does it export a game to retro platform like c64? keep on gaming

**Comment by haroldo-ok** [comment]
Hello, again;

Thanks for your interest! As of now, this tool only supports Sega Master System; I kinda intended to expand to other platforms as time went on, but that never materialized.

**Comment by zartan9** [comment]
Ah got it. You have so many creative platforms! I may revisit atarivcs builder you made. Can it have multiple "boxes" follow a player around the screen ..another idea of mine..

Sent from the all new AOL app for Android 
 
  On Sun, Jul 10, 2022 at 12:16 PM, Haroldo de Oliveira ***@***.***> wrote:   


Hello, again;

Thanks for your interest! As of now, this tool only supports Sega Master System; I kinda intended to expand to other platforms as time went on, but that never materialized.

**Comment by haroldo-ok** [comment]
Yes, it would be possible to have a few boxes following the player around, as long as it does not exceed Atari 2600 limits; for now, the tool only supports bBasic's default kernel, wich allows for two player sprites, two player missiles, and a ball. The player missiles and/or the ball can be used for representing rectangles, and you can change their X/Y coordinates to follow the player.

**Comment by zartan9** [comment]
thanks!


-----Original Message-----
